software/tracksight/frontend/components/EditModeToggle.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import { File, Pencil } from "lucide-react";
+
+import { useEditMode } from "@/lib/contexts/EditModeContext";
+
+const EditModeToggle = () => {
+  const { isEditMode, toggleEditMode } = useEditMode();
+
+  const Icon = isEditMode ? File : Pencil;
+
+  return (
+    <div className="hover:cursor-pointer">
```

Comment:
prefer a button here

Reply:
I just deleted this file as it's redundant with the new deisgn